src/plots/gl2d/scene2d.js

```diff
@@ -432,32 +433,13 @@ proto.plot = function(fullData, calcData, fullLayout) {
     this.mouseContainer.style.left = size.l + domainX[0] * size.w + 'px';
     this.mouseContainer.style.top = size.t + (1 - domainY[1]) * size.h + 'px';
 
-    var bounds = this.bounds;
-    bounds[0] = bounds[1] = Infinity;
-    bounds[2] = bounds[3] = -Infinity;
-
-    var traceIds = Object.keys(this.traces);
     var ax, i;
 
-    for(i = 0; i < traceIds.length; ++i) {
-        var traceObj = this.traces[traceIds[i]];
-
-        for(var k = 0; k < 2; ++k) {
-            bounds[k] = Math.min(bounds[k], traceObj.bounds[k]);
-            bounds[k + 2] = Math.max(bounds[k + 2], traceObj.bounds[k + 2]);
-        }
-    }
-
     for(i = 0; i < 2; ++i) {
-        if(bounds[i] > bounds[i + 2]) {
-            bounds[i] = -1;
-            bounds[i + 2] = 1;
-        }
```

Comment:
Also seems like `this.bounds` has stopped being used, so I 🔪 it. Can't wait until we can 🔪 this whole subplot type...